launch/personal_food_computer_var_types.yaml

```diff
@@ -0,0 +1,82 @@
+recipe_variables:
+  recipe_start:
+    name: recipe_start
+    description: "Represents the start of a recipe"
+  recipe_end:
+    name: recipe_end
+    description: "Represents the end of a recipe"
+environment_variables:
+  air_temperature:
+    name: air_temperature
+    description: "Temperature of the air in degrees Celcius"
+    units: "degrees C"
+  air_humidity:
+    name: air_humidity
+    description: "A measure of the concentration of water in the air relative to the maximum concentration at the current temperature"
+    units: "percent relative"
+  air_carbon_dioxide:
+    name: air_carbon_dioxide
+    description: "The amount of Carbon Dioxide in the air"
+    units: "ppm"
+  air_flush_on:
+    name: air_flush_on
+    description: "Turn on air flush (off by default)"
+    units: "boolean"
```

Comment:
@rwdavis513 you bring up a good point. All types are Float64 for all `environment` topics (booleans are encoded as 0 and 1). This is actually what caused the EWMA filter bug in the first place!

My intention is to have typed topics in future (std_msgs/Bool, std_msgs/Float64, etc), but I think that's too large of a change for this PR. Perhaps we can work around this by matching the var name(s) explicitly as we did before, then circle back in a follow-up PR to have true typed topics.